editor.planx.uk/src/@planx/components/Pay/Public/Pay.test.tsx

```diff
@@ -112,14 +133,56 @@ describe("Pay component when fee is undefined or £0", () => {
     expect(screen.queryByText("Continue")).not.toBeInTheDocument();
   });
 
-  it("Skips pay if fee = 0", () => {
+  it("Allows the user to view a fee breakdown and continue if the fee is zero", async () => {
     const handleSubmit = vi.fn();
 
     setState({ flow: flowWithZeroFee, breadcrumbs: breadcrumbs });
     expect(getState().computePassport()).toEqual({
       data: { "application.fee.payable": ["0"] },
     });
 
+    const { getByTestId, user, getByRole } = setup(
+      <Pay
+        title="Pay for your application"
+        fn="application.fee.payable"
+        handleSubmit={handleSubmit}
+        govPayMetadata={[]}
+      />,
+    );
+
+    // Node is not auto-answered
+    expect(handleSubmit).not.toHaveBeenCalled();
+
+    // Fee breakdown displayed
+    expect(getByTestId("fee-breakdown-table")).toBeVisible();
+
+    // User can continue
+    await user.click(getByRole("button", { name: "Continue" }));
+    expect(handleSubmit).toHaveBeenCalled();
+  });
+
+  it("Skips pay if fee is negative", () => {
+    const handleSubmit = vi.fn();
+    const loggerSpy = vi.spyOn(logger, "notify");
+
+    const negativeFeeBreadcrumbs: Breadcrumbs = {
+      setValue: {
+        auto: true,
+        data: {
+          "application.fee.payable": ["-12"],
+        },
+      },
+    };
+
+    setState({
+      flow: flowWithNegativeFee,
+      breadcrumbs: negativeFeeBreadcrumbs,
+    });
```

Comment:
Question: rather than auto-answer/skip in this scenario - why wouldn't we display (and log) an explicit error like "Something has gone wrong calculating your application fee" ? Similar to how we currently do for the `"undefined_fee"` state?  

Is there a risk to auto-answering, submitting, then the council having to manually prompt for payment and the applicant being oblivious to an issue? 

The error page could very well still have an enabled "Continue" button still, but at the least the applicant would be aware something has gone wrong and that they should _expect_ to be contacted about paying after the fact?

Reply:
In other words, should the Pay component _ever_ be auto-answered now?

Reply:
This is a great point - I've now updated this to behave the same as an undefined fee.